packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -77,7 +83,9 @@ export const buildModules = async (
     addResponseHeadersToConfigYaml(
       projectStructure,
       {
-        pathPattern: `^modules/${moduleName}.*`,
+        pathPattern: scope
```

Comment:
This shouldn't be dependent on scope, but where the module lives. If you define a scope but the module lives in root, this will output it to scope won't it?

Reply:
Ah good catch. I removed that portion entirely since in [SUMO-5833](https://yexttest.atlassian.net/browse/SUMO-5833) we don't want it there at all anyways.